doc/api/dgram.md

```diff
@@ -70,18 +70,19 @@ datagram messages. This occurs as soon as UDP sockets are created.
 added: v0.1.99
 -->
 
+The `'message'` event is emitted when a new datagram is available on a socket.
+The event handler function is passed two arguments: `msg` and `rinfo`.
 * `msg` {Buffer} - The message
 * `rinfo` {Object} - Remote address information
-
-The `'message'` event is emitted when a new datagram is available on a socket.
-The event handler function is passed two arguments: `msg` and `rinfo`. The
-`msg` argument is a [`Buffer`][] and `rinfo` is an object with the sender's
-address information provided by the `address`, `family` and `port` properties:
+  * `address` {String} The sender's address
+  * `family` {String} The sender's family
+  * `port` {Number} The sender's port
+  * `size` {Number} The message size
 
 ```js
 socket.on('message', (msg, rinfo) => {
-  console.log('Received %d bytes from %s:%d\n',
-              msg.length, rinfo.address, rinfo.port);
+  console.log('Received %d bytes from %s:%d using %s with a message size of %d\n',
+              msg.length, rinfo.address, rinfo.port, rinfo.family, rinfo.size);
 });
```

Comment:
Would be nice to show some example output in a comment afterwards, to justify the existence of the example. As-is, I think this so-called "example" offers exactly zero additional benefit over the docs, and would be better deleted. People already know how to write functions in js, and attach event listeners, which is all this shows.